src/main/java/Array14.java

```diff
@@ -0,0 +1,33 @@
+public class Array14 {
+    public int determinant(int array[][]) {
+        int det = 0;
```

Comment:
Luôn cố gắng minimal phạm vi của biến. Cụ thể ở đây, biến det nên được khai báo ngay phía trước vòng for

```suggestion
        int sum = 0;
```